README.md

```diff
@@ -1,21 +1,92 @@
 # Reviews Parsing MLSys
 
```

Comment:
Thiếu table of contents sếp ơi

Reply:
Em thấy Github README có sẵn ToC r mà sếp:
<img width="995" alt="image" src="https://github.com/user-attachments/assets/d882d892-22a2-433c-aa00-597580a820d5">

Reply:
Nhưng không biết mấy ông khác mà không phải Github có không ý sếp :v ok, cái này tùy sếp thui ạ.